dvc/env.py

```diff
@@ -1 +1,2 @@
 DVC_DAEMON = "DVC_DAEMON"
+DVC_PAGER = "DVC_PAGER"
```

Comment:
Random Q: What are these @efiop? Why not just use the string values instead of these constants?

Reply:
@jorgeorpinel Just older logic, the current agreement is to just use strings, but we've agreed with @xliiv to keep it consistent for now.